runtime/Cpp/runtime/src/Exceptions.cpp

```diff
@@ -22,3 +22,12 @@ IOException::IOException(const std::string &msg) : std::exception(), _message(ms
 const char* IOException::what() const NOEXCEPT {
   return _message.c_str();
 }
+
+IllegalStateException::~IllegalStateException() = default;
+IllegalArgumentException::~IllegalArgumentException() = default;
+NullPointerException::~NullPointerException() = default;
+IndexOutOfBoundsException::~IndexOutOfBoundsException() = default;
+UnsupportedOperationException::~UnsupportedOperationException() = default;
+EmptyStackException::~EmptyStackException() = default;
+CancellationException::~CancellationException() = default;
+ParseCancellationException::~ParseCancellationException() = default;
```

Comment:
OK, I see. In such cases make them empty d-tors in the cpp file - properly ordered with the rest of the code.